taichi/backends/vulkan/vulkan_utils.h

```diff
@@ -14,7 +14,7 @@ namespace vulkan {
 class VulkanEnvSettings {
  public:
   static constexpr uint32_t kApiVersion() {
-    return VK_API_VERSION_1_0;
+    return VK_API_VERSION_1_2;
```

Comment:
Why 1.2 after added submodule?

Reply:
Sorry for the back and forth lol... After https://github.com/taichi-dev/taichi/pull/2615 we can set Taichi's default Vulkan version to 1.2